docs/infrastructure.md

```diff
@@ -0,0 +1,82 @@
+## Infrastructure
+
+### [General Deployment Process Overview Doc](https://github.com/broadinstitute/dsp-devops-wiki/wiki/deployment_process)
+
+### Infrastructure Overview
+The Google Infrastructure is managed via Terraform and nearly all the kubenetes deployments and such are done via a Helm Chart. See [General Deployment Process Overview Doc](https://github.com/broadinstitute/dsp-devops-wiki/wiki/deployment_process) for more in depth guide. Datarepo is deployed on Kubernetes, Google Cloud Sql (postgres 11) and contains the following containers:
+
+- api
+- ui
+- open id connect proxy
+- google cloud sql proxy
+
+Container monitoring is done by [promethues operator](https://github.com/helm/charts/tree/master/stable/prometheus-operator) this includes:
+
+- alertmanager
+- grafana
+- state-metric
+- node-exporter
+- promethues
+
+Continuous deployment is managed by [Argocd](https://github.com/argoproj/argo-helm/tree/master/charts/argo-cd) this includes:
+- dex
+- argo-repo-server
+- argo-server
+- redis
+- argo-application-controller
+
+### Helm Overview
+#### Repos
+- [Datarepo-helm](https://github.com/broadinstitute/datarepo-helm)
+  - Contains datarepo specific charts api, ui, proxy
+- [monster-helm](https://github.com/broadinstitute/monster-helm)
+  - Contains gcp managed cert chart
+- [argo-helm](https://github.com/argoproj/argo-helm/tree/master/charts/argo-cd)
+  - Contains argocd chart for continuous deployment
+- [stable/prometheus-operator](https://github.com/helm/charts/tree/master/stable/prometheus-operator)
+  - Contains helm stable which includes the prometheus-operator chart
+
+#### Definitions
+This is specifically referring to a helm `Values.yaml` file. This file contains all the user specific values to be overritten in the chart being used. Datarepo uses an umbrella chart and each users values can be found [here](https://github.com/broadinstitute/datarepo-helm-definitions)
+
+## Github Actions
+GitHub Actions automates all your workflows and jobs that happen in the back ground. Here we will cover datarepo specific actions. These are mostly for internal use such as releasing charts and bumping deployments. This doc is going to assume you understand the basics [here](https://help.github.com/en/actions).
```

Comment:
I would mention somewhere that the way the GitHub Actions are driven now is by passing arguments to a main entrypoint which shells out to the appropriate job (as opposed to having everything in a single yaml).